bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -176,6 +176,9 @@ make_gridpack () {
       wget --no-check-certificate ${MGSOURCE}
       tar xzf ${MG}
       rm "$MG"
+
+      # agrohsje 
+      cp -rp ${PRODHOME}/addons/models/* ${MGBASEDIRORIG}/models/.
```

Comment:
this is not needed, no? @agrohsje

Reply:
I suppose it is not needed in general, though we use it when we make our EFT gridpacks.

Reply:
we already have a way to deal with model files (although not significantly different in terms of logic) through `_extramodels.dat`
this will also just complain if you don't have that directory, should remove this

Reply:
Removed